sncast_scripts/src/declare_presets.cairo

```diff
@@ -0,0 +1 @@
+mod declare_presets;
```

Comment:
Why can't we have the script code in here? Do we need the extra directory?

Reply:
Nope, updated!